agent/engine/daemonmanager/daemon_manager_windows.go

```diff
@@ -17,27 +17,15 @@
 package daemonmanager
 
 import (
-	"context"
-	"errors"
-
-	apitask "github.com/aws/amazon-ecs-agent/agent/api/task"
-	"github.com/aws/amazon-ecs-agent/agent/dockerclient/dockerapi"
-	"github.com/docker/docker/api/types"
+	"io/fs"
 )
 
-func (dm *daemonManager) CreateDaemonTask() (*apitask.Task, error) {
-	return nil, errors.New("daemonmanager.CreateDaemonTask not implemented for Windows")
-}
-
-func (dm *daemonManager) LoadImage(ctx context.Context, dockerClient dockerapi.DockerClient) (*types.ImageInspect, error) {
-	return nil, errors.New("dameonmanager.LoadImage not implemented for Windows")
-}
-
-// isImageLoaded uses the image ref with its tag
-func (dm *daemonManager) IsLoaded(dockerClient dockerapi.DockerClient) (bool, error) {
-	return false, errors.New("daemonmanager.IsLoaded not implemented for Windows")
-}
-
-func (dm *daemonManager) ImageExists() (bool, error) {
-	return false, errors.New("daemonmanager.ImageExists not implemented for Windows")
-}
+const (
+	daemonUID                                   = 0
+	daemonMountPermission           fs.FileMode = 0755
+	daemonLogPermission             fs.FileMode = 0777
+	ecsAgentLogFileENV                          = "ECS_LOGFILE"
+	defaultECSAgentLogPathContainer             = "/log"
+	socketPathHostRoot                          = "C:\\ProgramData\\Amazon\\ECS\\ebs-csi-driver"
```

Comment:
Please do not hardcode paths like this. Use https://github.com/aws/amazon-ecs-agent/blob/27ca59a0a052c2e8a6e330899f02969ba15dbe66/agent/config/config_windows.go#L86